### DISCUSSION TITLE
[Meta] Reorder the questions in the bug report template.

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#427] [Feature Request]
Right now, the bug report template has boxes in the following order:
* Describe the bug
* To reproduce
* Expected behavior
I have a habit of writing everything down in one set of text when writing issues / bug reports (most GH repos don't use the bug report template). Describe the bug and be renamed to "Describe any other details" and got below the other two.

**Comment by tukiminya** [comment]
The reason Floorp restricts the way Issues are written is out of necessity to reduce the burden on developers and to reduce the number of responses they receive about the environment they are using.
Since our native language is not English, it is difficult for us to understand the meaning properly and it is difficult to easily understand what is important and what is a clue if all the information is written as a sentence.
Also, most large communities like react, Next.js, MUI, Storybook and Gatsby, etc. have introduced "Issue template" or "Issue forms".
